common/ferrostar/src/models.rs

```diff
@@ -451,6 +451,18 @@ pub enum ManeuverModifier {
     SharpLeft,
 }
 
+/// The content of a visual instruction.
+#[derive(Debug, Clone, Eq, PartialEq)]
+#[cfg_attr(feature = "uniffi", derive(uniffi::Record))]
+#[cfg_attr(any(feature = "wasm-bindgen", test), derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "wasm-bindgen", serde(rename_all = "camelCase"))]
+#[cfg_attr(feature = "wasm-bindgen", derive(Tsify))]
+#[cfg_attr(feature = "wasm-bindgen", tsify(into_wasm_abi, from_wasm_abi))]
+pub struct LaneInfo {
+    pub active: bool,
+    pub directions: Vec<String>,
```

Comment:
I think `active_direction` might be worth adding too?

Reply:
done.